deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -8,7 +8,8 @@ mongo-password = {{ default .Env.mongo_password "" }}
 
 start-local-mongo = {{ default .Env.start_local_mongo "0" }}
 
-auth-service-url = {{ default .Env.auth_service_url "http://localhost" }}
+auth-service-url = {{ default .Env.auth_legacy_url "http://localhost" }}
+auth-url = {{ default .Env.auth_url "http://localhost" }}
 shock-url = {{ default .Env.shock_url "http://localhost" }}
 workspace-url = {{ default .Env.workspace_url "http://localhost" }}
```

Comment:
Why is there a ws url in the template? The HS shouldn't know WS exists.

Reply:
I think the service refused to start if there wasn't a ws url?